backend/src/main/java/ai/verta/modeldb/entities/ExperimentRunEntity.java

```diff
@@ -75,12 +75,6 @@ public ExperimentRunEntity(ExperimentRun experimentRun) throws InvalidProtocolBu
           RdbmsUtils.generateCodeVersionEntity(
               ModelDBConstants.CODE_VERSION, experimentRun.getCodeVersionSnapshot()));
     }
-
```

Comment:
was this not required?

Reply:
No, it is not required at that point. this removed code snippet was moved on DAO level because we have to populate the blob_type which is we can not fetch from ExperimentRunEntity OR we can not pass it as the parameter.